### PR TITLE
Ctrcri to ctr cri

### PR DESCRIPTION
**Pull request by Random-Liu**
Fixes https://github.com/containerd/cri/issues/590

This PR:
1) Upgrade containerd to https://github.com/containerd/containerd/commit/3c1ef1a714cf5b0104f340f76d539802fc24c75f.
2) Use `ctr cri load` instead.

### CODE REVIEW

**Comment by Random-Liu** [comment]
@mikebrow Can you take a look at this one?

File: cmd/ctr/main.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2018 The containerd Authors.
```
Comment (mikebrow) [inline cmd/ctr/main.go]: note: once #647 is pushed the year is no longer necessary. The containerd guys don't want the year.

File: pkg/version/version.go [hunk 2]
```diff
@@ -1,43 +0,0 @@
...
-import (
-	"fmt"
-
-	"github.com/blang/semver"
-)
-
-// CRIContainerdVersion is the version of the cri plugin.
-var CRIContainerdVersion = "UNKNOWN"
-
-func validateSemver(sv string) error {
```
Comment (mikebrow) [inline pkg/version/version.go]: does containerd already have a validateSemver unit test?

Comment (Random-Liu) [reply pkg/version/version.go]: No, they don't. I feel like this can be a CRI validation test instead. Unless kubelet stops relying on it.

Comment (Random-Liu) [reply pkg/version/version.go]: Filed an issue for this.
https://github.com/containerd/cri/issues/672

File: cmd/ctr/main.go (hunk 1 above)
Comment (Random-Liu) [reply cmd/ctr/main.go]: Will review that PR later. :P Need to finish the integration first. :)

**Pull request merged**